src/Epic/ContributionsEpicButtons.tsx

```diff
@@ -123,8 +124,10 @@ export const ContributionsEpicButtons = ({
     remindMeConfirmationText,
     remindMeConfirmationHeaderText,
     trackClick,
+    hidePaymentIcons,
 }: ContributionsEpicButtonsProps): JSX.Element => {
     const [sectionState, setSectionState] = useState<SectionState>('DEFAULT');
+    const showPaymentIcons = hidePaymentIcons != null && hidePaymentIcons === 'true' ? false : true;
```

Comment:
is this equivalent to `const showPaymentIcons = hidePaymentIcons !== 'true'`?

Reply:
Ha I made (almost) the exact same suggestion above :-) This is better though!

Reply:
Fixed using this alternative code